scripts/build_message_web.py

```diff
@@ -0,0 +1,146 @@
+import os, time, argparse
+import json
+from elasticsearch import Elasticsearch
+from datetime import datetime
+import classes.class_generate_events_json as calendar_events
+
+import support as support
+
+def fetch_current_indexed_sdk_files(es : Elasticsearch, index_name='github_live_index'):
+    # Search query to use
+    query_search = {
+        "size": 5000,
+        "query": {
+            "match_all": {}
+        }
+    }
+
+    # Search the base with provided query
+    num_of_retries = 1
+    while num_of_retries <= 10:
+        try:
+            response = es.search(index=index_name, body=query_search)
+            if not response['timed_out']:
+                break
+        except:
+            print("Executing search query - retry number %i" % num_of_retries)
+        num_of_retries += 1
+
+    all_packages = []
+    for eachHit in response['hits']['hits']:
+        if not 'name' in eachHit['_source']:
+            continue
+        if 'necto_package' == eachHit['_type']:
+            if False == eachHit['_source']['hidden']:
+                all_packages.append(eachHit['_source'])
+
+    return all_packages
+
+def fetch_current_indexed_click_boards(es : Elasticsearch, index_name='libstock_live_necto_cmake_packages'):
```

Comment:
Index name value as secret.

Reply:
Added it as ${{ secrets.ES_CLICKS_INDEX_LIVE }} - we need to add it to repo secrets.

Reply:
`ES_CLICKS_INDEX_LIVE` added to organization secrets.